src/keylessBackup/KeylessBackupProgress.tsx

```diff
@@ -257,15 +271,29 @@ function Restore() {
 
 function Setup({ origin }: { origin: KeylessBackupOrigin }) {
   const keylessBackupStatus = useSelector(keylessBackupStatusSelector)
+  const onboardingProps = useSelector(onboardingPropsSelector)
+  const { step, totalSteps } = getOnboardingStepValues(Screens.SignInWithEmail, onboardingProps)
   const { t } = useTranslation()
+  const { bottom } = useSafeAreaInsets()
+  const insetsStyle = {
+    paddingBottom: Math.max(0, 40 - bottom),
+  }
 
   const navigatedFromSettings = origin === KeylessBackupOrigin.Settings
+  const isOnboarding = origin === KeylessBackupOrigin.Onboarding
 
   const onPressContinue = () => {
     ValoraAnalytics.track(KeylessBackupEvents.cab_progress_completed_continue)
```

Comment:
maybe not in scope of this PR, but should this include origin?

Reply:
Since it's within the setup function I don't think we need to specify the `KeylessBackupFlow`.

Reply:
origin can still be onboarding / settings right?

Reply:
Added the origin to the event in f54214c8af1b190924092e3eb80d056494b78e3f & updated the tests in 779f0172358734d92481ac22a6db0db25f54394e!